### PR TITLE
chore: add video-temporal-localization

### PR DESCRIPTION
**Pull request by CamiloInx**
Description
---
* Add the video temporal localization tool to vision-agent
* Add test

### CODE REVIEW

File: vision_agent/tools/tools.py [hunk 1]
```diff
@@ -2162,6 +2210,7 @@ def _plot_counting(
     generate_pose_image,
     closest_mask_distance,
     closest_box_distance,
+    video_temporal_localization,
```
Comment (dillonalaird) [inline vision_agent/tools/tools.py:2213]: Just remove this line, this is what adds it to the VisionAgent.

**Review by dillonalaird — CHANGES_REQUESTED** [review]
Looks good! Just remove the tool from the FUNCTION_TOOLS list

File: vision_agent/tools/tools.py (hunk 1 above)
Comment (CamiloInx) [reply vision_agent/tools/tools.py]: Done!

**Review by dillonalaird — APPROVED** [review]
LGTM

**Pull request merged**